src/main/java/io/camunda/testing/assertions/MessageAssert.java

```diff
@@ -119,15 +120,17 @@ public MessageAssert hasNotExpired() {
     return this;
   }
 
-  public ProcessInstanceAssert extractingProcessInstance() {
+  /**
+   * Extracts the process instance for the given message
+   *
+   * @param isMessageStartEvent is the message a start event
+   * @return Process instance assertions {@link ProcessInstanceAssert}
+   */
+  public ProcessInstanceAssert extractingProcessInstance(final boolean isMessageStartEvent) {
```

Comment:
:question: Is there some way we can get rid of the parameter? Can we somehow figure that out by looking at both record types and picking the one that matches?